dissect/util/feature.py

```diff
@@ -0,0 +1,66 @@
+import functools
+import os
+from enum import Enum
+from typing import Callable, Optional
+
+
+# Register feature flags in a central place to avoid chaos
+class Feature(Enum):
+    NOVICE = "novice"
+    LATEST = "latest"
+    BETA = "beta"
+
+
+# This set defines the valid flags
+DISSECT_FEATURE_SET = set(item for item in Feature)
+
+# Defines the default flags (as strings)
+DISSECT_FEATURES_DEFAULT = "novice/latest"
```

Comment:
And this could be an OR of the novice and latest flag.

Reply:
See comment above.